www/main/overlay/components/LayoutSelector.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LayoutSelector = ({ changeLayout }) => {
+  const layouts = ['top', 'bottom', 'split', 'vertical', 'classic'];
+  const layoutMarkup = layouts.map(layout => (
+    <div
+      key={`layout-${layout}`}
+      className={`layout-btn ${layout}`}
+      data-layout={layout}
```

Comment:
can this attribute be camelCase?

Reply:
I dont think so, saw this documentation link
https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html#data-and-aria-attributes